client/galaxy/scripts/components/User/RecentInvocations.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <invocations
+        :invocationItems="invocationItems" 
+        :loading="loading"
+        :busy="busy"
+        headerMessage="Your most recent workflow invocations are displayed on this page."
+        noInvocationsMessage="There are no workflow invocations to show.">
+    </invocations>
+</template>
+
+<script>
+import Invocations from "components/admin/Invocations";
+import { getRecentInvocations } from "./UserServices";
+
+export default {
+    components: {
+        Invocations
+    },
+    data() {
+        return {
+            invocationItems: [],
+            loading: true,
+            busy: true,
```

Comment:
Wondering if and how we use the `busy` attribute here and if it is equivalent to `loading`.

Reply:
Yeah - eliminated that - good point!